manticore/core/cpu/abstractcpu.py

```diff
@@ -477,6 +478,74 @@ def read_bytes(self, where, size):
             result.append(Operators.CHR(self.read_int( where+i, 8)))
         return result
 
+    def read_string(self, where, max_length=None):
+        '''
+        Read a NUL-terminated concrete buffer from memory.
+
+        :param int where: Address to read string from
+        :param int max_length:
+            The size in bytes to cap the string at, or None [default] for no
+            limit.
+        :return: string read
+        :rtype: str
+        '''
+        s = StringIO.StringIO()
+        while True:
+            c = self.read_int(where, 8)
```

Comment:
This function is technically defined to work solely on concrete buffers. What would be the correct behavior? Implement it via Mark's `find_zero`?